modules/EvseManager/EvseManager.cpp

```diff
@@ -1175,14 +1175,15 @@ void EvseManager::setup_v2h_mode() {
     types::energy::ExternalLimits external_limits;
     types::energy::ScheduleReqEntry target_entry;
     target_entry.timestamp = timestamp;
-    target_entry.limits_to_leaves.total_power_W = powersupply_capabilities.max_import_power_W.value();
+    target_entry.limits_to_leaves.total_power_W = {powersupply_capabilities.max_import_power_W.value_or(0.),
```

Comment:
`.value_or(0.)` should not actually be required, as it is already checked beforehand whether `max_import_power_W` has a value.